<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1871VKV" ignored="1" name="Aaything _add_Test" tests="30" failures="6" timestamp="2021-05-08T11:11:22 IST" time="19.59" errors="0">
  <testcase name="UserLogin" time="0.011" classname="besic_learing_testgn.method_ex"/>
  <testcase name="UserMoneyTranfar" time="0.002" classname="besic_learing_testgn.method_ex"/>
  <testcase name="Begin" time="0.007" classname="besic_learing_testgn.Alwaysrun">
    <failure type="java.lang.AssertionError" message="we are fail this task case">
      <![CDATA[java.lang.AssertionError: we are fail this task case
at besic_learing_testgn.Alwaysrun.Begin(Alwaysrun.java:14)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Begin -->
  <testcase name="Ready" time="0.002" classname="besic_learing_testgn.Alwaysrun"/>
  <testcase name="Steady" time="0.002" classname="besic_learing_testgn.Alwaysrun"/>
  <testcase name="Go" time="0.0" classname="besic_learing_testgn.Alwaysrun">
    <skipped/>
  </testcase> <!-- Go -->
  <testcase name="Criticals" time="0.003" classname="besic_learing_testgn.Assertions">
    <failure type="java.lang.AssertionError" message="expected [We are learninh in English] but found [We are learing in englis and Hindi]">
      <![CDATA[java.lang.AssertionError: expected [We are learninh in English] but found [We are learing in englis and Hindi]
at besic_learing_testgn.Assertions.Criticals(Assertions.java:13)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Criticals -->
  <testcase name="High" time="0.006" classname="besic_learing_testgn.Assertions"/>
  <testcase name="Low" time="0.002" classname="besic_learing_testgn.Assertions"/>
  <testcase name="Medium" time="0.006" classname="besic_learing_testgn.Assertions">
    <failure type="java.lang.AssertionError" message="Fail deliberately ">
      <![CDATA[java.lang.AssertionError: Fail deliberately 
at besic_learing_testgn.Assertions.Medium(Assertions.java:27)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Medium -->
  <testcase name="DataMatrix" time="0.004" classname="besic_learing_testgn.Data_provider"/>
  <testcase name="DataMatrix" time="0.004" classname="besic_learing_testgn.Data_provider"/>
  <testcase name="DataMatrix" time="0.002" classname="besic_learing_testgn.Data_provider"/>
  <testcase name="SetUp" time="0.007" classname="besic_learing_testgn.ITest_Context"/>
  <testcase name="Logic" time="0.004" classname="besic_learing_testgn.ITest_Context"/>
  <testcase name="exit" time="0.009" classname="besic_learing_testgn.ITest_Context"/>
  <testcase name="Register_User" time="0.002" classname="besic_learing_testgn.Priority_Enabled"/>
  <testcase name="Valid_Login" time="0.002" classname="besic_learing_testgn.Priority_Enabled"/>
  <testcase name="Home_Page" time="0.007" classname="besic_learing_testgn.Priority_Enabled"/>
  <testcase name="User_LogOut" time="0.002" classname="besic_learing_testgn.Priority_Enabled"/>
  <testcase name="HartAssertLogic" time="0.002" classname="besic_learing_testgn.HartAssert_Vs_SoftAssert">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at besic_learing_testgn.HartAssert_Vs_SoftAssert.HartAssertLogic(HartAssert_Vs_SoftAssert.java:15)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- HartAssertLogic -->
  <testcase name="SoftAssertLogic" time="0.013" classname="besic_learing_testgn.HartAssert_Vs_SoftAssert">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at besic_learing_testgn.HartAssert_Vs_SoftAssert.SoftAssertLogic(HartAssert_Vs_SoftAssert.java:35)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- SoftAssertLogic -->
  <testcase name="landing_page" time="0.069" classname="besic_learing_testgn.Dependencies_ex"/>
  <testcase name="calculater" time="6.966" classname="besic_learing_testgn.Dependencies_ex"/>
  <testcase name="ReturnToHome" time="0.247" classname="besic_learing_testgn.Dependencies_ex"/>
  <testcase name="tearDown" time="1.49" classname="besic_learing_testgn.Dependencies_ex"/>
  <testcase name="loginTest" time="0.002" classname="besic_learing_testgn.TestGN_annotations"/>
  <testcase name="Validation" time="0.003" classname="besic_learing_testgn.Soft_Assertions">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	B2 is wrong string statement expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
B2 is wrong string statement expected [true] but found [false]
at besic_learing_testgn.Soft_Assertions.Validation(Soft_Assertions.java:22)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Validation -->
  <testcase name="loginTest" time="0.002" classname="besic_learing_testgn.TestGN_annotations2_with2testcase"/>
  <testcase name="registerTest" time="0.002" classname="besic_learing_testgn.TestGN_annotations2_with2testcase"/>
  <testcase name="AddNews" time="0.0" classname="besic_learing_testgn.Priority_Enabled">
    <ignored/>
  </testcase> <!-- AddNews -->
</testsuite> <!-- Aaything _add_Test -->
